components/basecomponents/LanguageSelect.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ButtonHTMLAttributes, forwardRef } from 'react';
+import ChevronRight from '../../public/icons/chevron-right.svg';
+
+interface LanguageSelectProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  active: boolean;
+  languages: [string, string];
+}
+
+export const LanguageSelect = forwardRef((props: LanguageSelectProps, ref: React.ForwardedRef<HTMLButtonElement>): JSX.Element => {
+  const { active, languages, ...rest } = props;
+  return (
+    <button
+      aria-selected={active}
```

Comment:
Eslint complains here: "The attribute aria-selected is not supported by the role button. This role is implicit on the element button." 

It seems defining the button role as "option" gets rid of the warning. @petrillek is that a correct way to resolve this? Thanks :)

```suggestion
      aria-selected={active}
      role="option"
```